lib/generators/scenic/view/templates/db/migrate/create_view.erb

```diff
@@ -1,5 +1,5 @@
 class <%= migration_class_name %> < ActiveRecord::Migration
   def change
-    create_view :<%= plural_name %><%= ", materialized: true" if materialized? %>
+    create_view "<%= plural_name %>"<%= ", materialized: true" if materialized? %>
```

Comment:
Yup, my suggestion is that we keep symbols for the normal case and use strings otherwise, so a `recent_hires` view would generate `create_view :recent_hires` but a `schema.recent_hires` would generate `create_view "schema.recent_hires"`.